apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -58,6 +58,11 @@ export const PropertyIcon = forwardRef<
         ref={forwardRef}
         size="1"
         css={{
+          // @note DO NOT REMOVE WITHOUT REPLACING WITH TOKENS
```

Comment:
The idea Andrey and I are pushing is to actually not do the nudges at all and instead fix it later properly with changes in the scales etc, so the preference is towards have small suboptimal visual things instead of heaving these